src/mol-model-formats/structure/property/bonds/atom.ts

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
Could you move this to property/atoms/chem_comp.ts and also rename property/bonds/comp.ts to chem_comp.ts?

Reply:
Done in 58e0ba0